CPS-XXXX/README.md

```diff
@@ -0,0 +1,134 @@
+---
+CPS: XXXX
+Title: Node Behavior during Low Participation
+Status: Draft
+Category: Consensus
+Authors:
+    - Nicolas Frisby <nick.frisby@iohk.io> <nicolas.frisby@moduscreate.com>
+Proposed Solutions: []
+Discussions:
+    - https://github.com/cardano-foundation/CIPs/pull/982
+Created: 2025-02-04
+License: CC-BY-4.0
+---
+
+## Abstract
+
+It is unclear how the Cardano node should behave when the best chain it has access to is so sparse that it violates the Praos security argument's lower bound on Chain Growth.
+This CPS motivates that question, lists some relevant concerns, and hopefully attracts some useful CIPs.
+
+The [Praos security argument](https://eprint.iacr.org/2017/573) assigns overwhelming probability to a lower bound on growth under the assumption that all honest parties are participating and together control more stake than the adversary.
+For Cardano specifically, that Chain Growth property has been instantiated as at least 2160 blocks of growth during each 36 hr interval, whereas the expected Cardano growth is 2160 blocks per 12 hr (assuming full participation).
+On the other hand, the [Disaster Recovery Plan](https://github.com/cardano-foundation/CIPs/tree/master/CIP-0135), intervenes when the chain does not grow at all for 36 hours.
+These two specifications do not address an observed growth rate less than 2160 per 36 hr but greater than zero, nor does any other specification.
+
+Such low growth rates become plausible when the assumptions of the Praos argument are violated for an extended period of time.
+Global-scale disasters, infrastructure attacks, local outages, eclipse attacks, bugs, negligent misconfiguration, etc could all cause it.
+
+## Problem
+
+The following mutually exclusive options seem to cover the design space, but each has unique advantages and disadvantages.
+
+- *AssumeImmutableAge* (AssumeIA).
+  The node will not switch to a chain if doing so would rollback more than 2160 blocks regardless of their age.
+  This limit is justified by the Common Prefix property of the Praos security argument as instantiated for Cardano.
+  (Note the absence of the Chain Growth property.)
+
+- *EnforceImmutableAge* (EnforceIA).
+  As a refinement of AssumeIA, the node will not switch to a chain if doing so would rollback more than 2160 blocks and/or a block that is more than 36 hr old (TODO: should age be with respect to the wall clock or the selection's tip?).
+  This limit is a disjunction of Common Prefix and Chain Growth.
+
+- *EnforceChainGrowth* (EnforceCG).
+  In addition to AssumeIA, the node rejects any chain that has less than 2160 blocks per 36 hr (even if selecting it wouldn't require rolling back any blocks).
```

Comment:
It's not directly clear to the reader (or, this reader at least!) how these immediately relate to the question above, since they don't say "what does the node do if it has a sparse chain" but rather constrain some other operations. I would suggest moving the *TwoThirdsInaccessible* block up above the description of the options, in order to outline the things that the node _could do_, and then from there lead into the possibilities and how they relate to various security arguments

Reply:
I added an intro and then added some connections between these options and the intro. Please let me know if that suffices. So far, I'm hesitant to introduce the scenarios first, but maybe that'd be better.